backend/ibutsu_server/controllers/dashboard_controller.py

```diff
@@ -53,10 +57,14 @@ def get_dashboard_list(project_id=None, user_id=None, page=1, page_size=25):
     :rtype: DashboardList
     """
     query = Dashboard.query
-    if project_id:
+    if "project_id" in connexion.request.args:
+        project = Project.query.get(connexion.request.args["project_id"])
+    if project:
```

Comment:
Can `project` be undefined here? Or will project_id always be in the request args?

Reply:
Ah yes, that needs to be fixed.

Reply:
Hrm, yes, there's a potential for project to be undefined. Let me fix that.